pkg/apis/core/v1alpha1/cluster_types.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2020 The Tilt Dev Authors
```

Comment:
```suggestion
Copyright 2021 The Tilt Dev Authors
```
old copyright template. .emacs aside: `(add-hook 'write-file-hooks 'copyright-update)` 😉